lib/phoenix/endpoint.ex

```diff
@@ -533,6 +533,24 @@ defmodule Phoenix.Endpoint do
     end
   end
 
+  @doc false
+  # This is private because it's really just a "hack" to instrument with an
```

Comment:
We can make this public. It is alright. We document it is useful when you want to dynamically dispatch to an instrumentation endpoint.